velox/docs/functions/spark/math.rst

```diff
@@ -0,0 +1,57 @@
+====================================
+Mathematical Functions
+====================================
+
+.. spark:function:: abs(x) -> [same as x]
+
+    Returns the absolute value of ``x``.
+
+.. spark:function:: add(x, y) -> [same as x]
+
+    Returns the result of adding x to y. The types of x and y must be the same.
+    For integral types, overflow results in an error. Corresponds to sparks's operator ``+``.
+
+.. spark:function:: ceil(x) -> [same as x]
+
+    Returns ``x`` rounded up to the nearest integer.  
+    In spark only long, double, and decimal have ceil.
```

Comment:
spark -> Spark.

> In spark only long, double, and decimal have ceil.

Supported types are: BIGINT, DOUBLE and DECIMAL.

Would you double check that decimal types are indeed supported in our implementation?

Reply:
Yes, I checked. decimal is not supported now [here](https://github.com/facebookincubator/velox/blob/main/velox/functions/sparksql/RegisterArithmetic.cpp#L48-L53).

Reply:
Thank you for checking. Let's make sure the documentation matches the implementation.